lib/modules/manager/github-actions/parse.ts

```diff
@@ -0,0 +1,290 @@
+import is from '@sindresorhus/is';
+import { regEx } from '../../../util/regex';
+
+function splitFirstFrom(
+  str: string,
+  sep: string,
+  start: number,
+): [string, string] | null {
+  const idx = str.indexOf(sep, start);
+  if (idx === -1) {
+    return null;
+  }
+  return [str.slice(0, idx), str.slice(idx + sep.length)];
+}
+
+function splitFirst(str: string, sep: string): [string, string] | null {
+  return splitFirstFrom(str, sep, 0);
+}
+
+interface QuotedValue {
+  value: string;
+  quote: string;
+}
+
+export function parseQuote(input: string): QuotedValue {
+  const trimmed = input.trim();
+  const first = trimmed[0];
+  const last = trimmed[trimmed.length - 1];
+
+  if (
+    trimmed.length >= 2 &&
+    first === last &&
+    (first === '"' || first === "'")
+  ) {
+    return { value: trimmed.slice(1, -1), quote: first };
+  }
+
+  return { value: trimmed, quote: '' };
+}
+
+/**
+ * Docker container:
+ * - `docker://image:tag`
+ * - `docker://image@digest`
```

Comment:
```suggestion
 * - `docker://image@digest`
 * - `docker://image:tag@digest`
```
what about that common usage?